code/game/objects/objs.dm

```diff
@@ -242,3 +244,37 @@
 		current_skin = choice
 		icon_state = unique_reskin[choice]
 		to_chat(M, "[src] is now skinned as '[choice].'")
+
+/obj/proc/can_be_unfasten_wrench(mob/user, silent) //if we can unwrench this object; returns SUCCESSFUL_UNFASTEN and FAILED_UNFASTEN, which are both TRUE, or CANT_UNFASTEN, which isn't.
```

Comment:
It's already obj/proc/unfasten. I'm just moving the proc to it's correct location in the file directory, see /code/game/machinery/_machinery.dm in the changes.

I can change it to only affect machinery, but it would likely mess up some structures or other things.

Reply:
tbh it's just copypaste it's fine

Reply:
my reading comprehension is utter shite